### PR TITLE
feat: add Marketing Campaign Manager role to jobs

### PR DESCRIPTION
**Pull request by marneysof**
## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
      and signed the CLA.
- [x] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [x] I have added tests that prove my fix is effective or that my feature
      works.
- [x] I have added necessary documentation within the code base (if
      appropriate).

### CODE REVIEW

File: src/markdown/jobs/marketing-campaign-manager.mdx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+- **Position Type:** Full Time
+- **Location**: Munich, Germany
+- **Components**: Google Suite, Slack, Github
+
+### What You'll Do
+
+- Devise and execute marketing campaigns to expand Ory’s reach among prospects and customers across multiple channels
+- Create and manage a marketing campaign calendar
+- Assist team workflow, delegating tasks and communicating across different teams
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/marketing-campaign-manager.mdx:38]:
```suggestion
- Assist team workflow and communicate across different teams
```

File: src/markdown/jobs/marketing-campaign-manager.mdx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+- Collaborate with external creative agencies when necessary
+- Research competitor strategies and industry standards in brand and performance campaigns
+- Benchmark innovative and flexible content creation and campaign management workflows
+- Make creative, engaging, and impactful marketing content in collaboration with team members
+- Grow and shape Ory’s presence as global brand with open source roots
+- Explore new channels for reaching and galvanizing relevant audiences
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/marketing-campaign-manager.mdx:44]:
```suggestion
- Explore new channels for reaching relevant audiences for Ory Cloud
```

**Review by BlueAcidFrog — DISMISSED** [review]

**Comment by aeneasr** [comment]
Good to go @BlueAcidFrog ?

**Comment by BlueAcidFrog** [comment]
@aeneasr , yes, good to go :)

**Review by BlueAcidFrog — APPROVED** [review]

**Review by aeneasr — APPROVED** [review]

**Pull request merged**